src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a
+        href="./index.html"
+        class="header_logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header-nav">
+        <ul class="nav-list">
+          <li class="nav-list_item">
+            <a
+              href="#"
+              class="nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav-list_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav-list_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
```

Comment:
HTML Content: Don't use spaces in `<a>` tag's `href` property. Anchor links starts with the `#` symbol.